### PR TITLE
[Automation] script bundle_demos get demos data [1.7.x]

### PR DESCRIPTION
**Pull request by daniels290813**
https://iguazio.atlassian.net/browse/IG-18569

### CODE REVIEW

File: automation/scripts/bundle_demos.py [hunk 1]
```diff
@@ -51,6 +65,21 @@
 # Copying update_demos.sh from mlrun
 shutil.copy2("automation/scripts/update_demos.sh", "demos/update_demos.sh")
 
+for demo, files in demos_files.items():
+    print(f"Downloading {len(files)} files for demo : {demo}.")
```
Comment (gtopper) [inline automation/scripts/bundle_demos.py]: Minor suggestion.
```suggestion
    print(f"Downloading {len(files)} files for demo {demo}...")
```

File: automation/scripts/bundle_demos.py [hunk 2]
```diff
@@ -51,6 +65,21 @@
...
+        os.makedirs(os.path.join(temp_dir, "data", demo), exist_ok=True)
+        print(f"Downloading {url}...")
+        # Download the file and save it
+        try:
+            response = requests.get(url)
+            with open(file_name, "wb") as f:
+                f.write(response.content)
+        except Exception as e:
+            print(f"could not download {url}")
+            print(e)
```
Comment (gtopper) [inline automation/scripts/bundle_demos.py]: Might want to `traceback.print_exc()` to print the error type and trace as well and not just the error message.

Comment (daniels290813) [reply automation/scripts/bundle_demos.py]: tnx @gtopper !

File: automation/scripts/bundle_demos.py [hunk 3]
```diff
@@ -51,6 +66,22 @@
...
```
Comment (gtopper) [inline automation/scripts/bundle_demos.py]: You can remove the print. The message is already printed by `traceback.print_exc()`.
```suggestion
```

**Review by aviaIguazio — APPROVED** [review]

**Review by liranbg — APPROVED** [review]

**Pull request merged**